scripts.py

```diff
@@ -795,21 +795,21 @@ def process_lane(self, lane, exporter=None):
         # integration.
         integration_link = get_one(
             self._db, ExternalIntegrationLink,
-            other_integration_id=exporter.integration.id,
+            external_integration_id=exporter.integration.id,
```

Comment:
The fact that you changed this without needing to change a test tells me there's a problem in test_scripts.py:TestCacheMARCFiles.test_process_lane, I think the culprit is this code, which (if you look at it in context) is creating a link to itself:

```
integration_link = self._external_integration_link(
            integration=integration,
            other_integration=exporter.integration,
            purpose=ExternalIntegrationLink.MARC
        )
```
Can you change the test file so that it sets up a more realistic link from a MARC export integration to a storage integration? Then this code will be properly tested.

Reply:
This test does not exercise the real `MARCExporter.records` and doesn't really use functionality of the `other_integration`, so the `other_integration` here just needs to be some `ExternalIntegration`. That said, making it something different certainly makes it more clear what _is_ being tested.